tests/safeds/data/tabular/transformation/test_imputer.py

```diff
@@ -120,7 +122,13 @@ def test_should_raise_if_column_not_found(self, strategy: ImputerStrategy) -> No
             },
         )
 
-        transformer = Imputer(strategy).fit(table_to_fit, None)
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                action="ignore",
+                message=r"There are multiple most frequent values in a column given to the Imputer\..*",
+                category=UserWarning,
+            )
+            transformer = Imputer(strategy).fit(table_to_fit, None)
```

Comment:
```suggestion
        if isinstance(strategy, Imputer.Strategy.Mode):
            with warnings.catch_warnings():
                warnings.filterwarnings(
                    action="ignore",
                    message=r"There are multiple most frequent values in a column given to the Imputer\..*",
                    category=UserWarning,
                )
                transformer = Imputer(strategy).fit(table_to_fit, None)
        else:
            transformer = Imputer(strategy).fit(table_to_fit, None)
```
Only filter the warning if you use the `Mode` strategy